ImageD11/sinograms/dataset.py

```diff
@@ -20,6 +20,8 @@
 - indexing and reconstructing grains
 
 """
+
+POSSIBLE_DETECTOR_NAMES = ("frelon3", "eiger")
```

Comment:
We should just read the name from the dataset. There should be only one entry which has:

/scan/measurement/detector -> attrs[ "interpretation" ] == "image"